docs/association/legal/Privacy_policy.md

```diff
@@ -1,63 +1,90 @@
-# Aragon Privacy Policy
+# Privacy Policy
 
-*Aragon contact details: contact@aragon.one*
+This document sets the privacy rules concerning the use of Aragon Association (the “Aragon”) sites and services, including 
 
-*Mail for requests and queries on privacy: contact@aragon.one*
+https://aragon.org;  
+https://mainnet.aragon.org;  
+https://rinkeby.aragon.org;
+https://forum.aragon.org;  
+https://wiki.aragon.org;  
+https://anj.aragon.org;  
+https://transparency.aragon.org  
```

Comment:
Perhaps we should just use a wildcard for any aragon.org subdomain? We're already forgetting some here (e.g. poweredby, convert) :).

Reply:
@sohkai good catch, that was actually a comment on the original draft but looks like it was overlooked or otherwise not included in the final copy. I had originally suggested: "perhaps can use `aragon.org and all subdomains` as a catchall?"
cc @joeycharlesworth

Reply:
I think that would make sense. @josesousapinto does that sound like a good idea? If so, please make the relevant edit.

Reply:
Yes, sounds good to me. One question: what should the contact email be for privacy matters? We should keep it to just one, for all Aragon privacy issues. In the privacy policy is mentioned privacy@aragon.org., but on the document edited is contact.@aragon.one. Can we keep just the aragon.org general email for that?

Reply:
hey @josesousapinto in case it is not clear, this is the version of the policy you should be looking at:

https://github.com/aragon/aragon-wiki/blob/4e0d507802e4411ba8bfa0d254ed3371fe330f8a/docs/association/legal/Privacy_policy.md

if you go to the top of this PR and click on the "Files changed" tab, then this new version will be on the right with the new parts highlighted in green, and the old version will be on the left with the parts that have been replaced highlighted in red.

Reply:
Updated per additional comments left on this thread:

https://github.com/aragon/aragon-wiki/commit/c7cc7d09c7e60d012c09b34c29bf6bd7b7b81531

Thank you @josesousapinto @joeycharlesworth and @sohkai for additional review!